code/datums/ai/basic_mobs/basic_subtrees/prepare_travel_to_destination.dm

```diff
@@ -0,0 +1,23 @@
+
+///Subtree that checks if we are on the target atom's tile, and sets it as a travel target if not
+///The target is taken from the blackboard. This one always requires a specific implementation.
+/datum/ai_planning_subtree/prepare_travel_to_destination
+	var/target_key
+	var/travel_destination_key = BB_TRAVEL_DESTINATION
```

Comment:
In fact we already have it as an argument on the subtree, so if you changed this you would be confused by it not also changing in the behaviour